### PR TITLE
[8.0] mis_builder

### PR DESCRIPTION
**Pull request by sbidoul**
This is the mis_builder PR.

Some examples reports are in the test data and in the module mis_builder_demo.
See also http://www.slideshare.net/acsone/misbuilder (slides show at Odoo Experience 2015).

The code is production ready but there are a few things left to do:
- [x] replace Acsone icon
- [x] README.rst 
- [ ] increase test coverage

### CODE REVIEW

**Comment by gurneyalex** [comment]
is this still WIP? if not, can you change the title of the PR?

**Comment by sbidoul** [comment]
@gurneyalex I rebased, added a README and screenshots and replaced the acsone icon by the OCA icon. 

Except for the test coverage, this is not WIP anymore. It works fine and reviews are welcome.

**Comment by gurneyalex** [comment]
@sbidoul thanks!

Runbot is currently red (orange actually, but on this side anything not green is red...) because of the following warning:

```
WARNING  openerp.addons.base.module.module: module mis_builder_demo: description is empty 
```

which is probably something easy to fix

File: mis_builder/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+##############################################################################
+
+{
+    'name': 'MIS Builder',
+    'version': '0.2',
+    'category': 'Reporting',
+    'summary': """
+        Build 'Management Information System' Reports and Dashboards
+    """,
+    'author': 'ACSONE SA/NV',
```
Comment (gurneyalex) [inline mis_builder/__openerp__.py]: can you add "Odoo Community Association (OCA)" to the authors?

**Comment by gurneyalex** [comment]
globally :+1: once the 2 reported issues are fixed. 

Thanks @sbidoul this is a great contribution to OCA!

**Comment by alexis-via** [comment]
I made a small PR to acsone's branch https://github.com/acsone/account-financial-reporting/pull/6

I'm surprised that Travis is green, because I've got some flake8 warnings here (and I'm supposed to have the same flake8 config as OCA...):

16:32 alexis@silence ~/tmp/account-financial-reporting/mis_builder % flake8 .
./models/aep.py:295:80: E501 line too long (80 > 79 characters)

It's just that I'm surprised, I don't care about this kind of stupid flake8 warnings. I fixed this warning in my PR too... but, as Travis is green, this part may not be necessary...

**Comment by lmignon** [comment]
@gurneyalex Changes done.

**Comment by alexis-via** [comment]
:+1: A great module, works well.
If my PR https://github.com/acsone/account-financial-reporting/pull/6 is accepted, it will be even better :)

**Comment by sbidoul** [comment]
I integrated @alexis-via's https://github.com/acsone/account-financial-reporting/pull/6 and slightly improved it.

I think all current review comments have been handled.

**Comment by alexis-via** [comment]
Yes, let's merge it !

By the way, I started working on l10n_fr_mis_reports... inspired by l10n_be_mis_reports.

**Comment by pedrobaeza** [comment]
Let me check this afternoon. This one is one of my pendings from several weeks.

**Comment by pedrobaeza** [comment]
@sbidoul, congratulations, you've made an excellent work! In first tests, it seems that this module can replace the one we are using for balance reporting (https://github.com/OCA/l10n-spain/tree/8.0/account_balance_reporting). Although there's work to do, and the UI is a bit uglier than ours (because it has more possibilities), on v9 we will make the switch.

Thanks for bringing this excellent work :+1: 

I merge finally this PR.

**Comment by sbidoul** [comment]
Thank you @pedrobaeza 

> Although there's work to do, and the UI is a bit uglier than ours (because it has more possibilities)

If you have specific ideas for improvements, do not hesitate to create an issue or a wiki page listing them so we can take them into account in future thinking on this topic.

**Comment by pedrobaeza** [comment]
I was thinking about specific UI customization to make smooth the transitioning from current to this module. For example, spanish balance reporting always required a comparison between current fiscal year and the previous one. This was automatically done in current module. Here, you have to define the time intervals. Maybe a wizard can do the trick.

**Pull request merged**